pkg/cmd/auth/setupgit/setupgit.go

```diff
@@ -79,7 +81,13 @@ func setupGitRun(opts *SetupGitOptions) error {
 
 	stderr := opts.IO.ErrOut
 	cs := opts.IO.ColorScheme()
-
+	if opts.Force && opts.Hostname == "" {
```

Comment:
Most of the times when we do flag validation, we do it in the cobra related code, in the `RunE` and then we return a `cmdutil.FlagErrorf`

Reply:
See https://github.com/cli/cli/blob/trunk/pkg/cmd/alias/delete/delete.go#L36 for an example

Reply:
This was something new for me 👍